nion/swift/ExportDialog.py

```diff
@@ -246,66 +250,176 @@ def cancel(self) -> bool:
         return True
 
 
-class ExportSVGHandler:
-    def __init__(self, display_size: Geometry.IntSize) -> None:
-        self.width_model = Model.PropertyModel(display_size.width)
-        self.height_model = Model.PropertyModel(display_size.height)
+class UnitType(enum.Enum):
+    PIXELS = 0
+    INCHES = 1
+    CENTIMETERS = 2
+
+
+ConversionUnits = {
+    UnitType.PIXELS: 1,
+    UnitType.CENTIMETERS: 37,
```

Comment:
These should be floating point values. The conversion to pixels for centimeters results in a ~2% error in the size when using "37" for the pixels per centimeter (which is literally defined several lines above) instead of the more accurate `PIXELS_PER_CM = 37.795275591` value.